azure-pipelines-harrastukset-review.yml

```diff
@@ -54,3 +54,20 @@ extends:
   # Django example: azure-pipelines-PROJECTNAME-api-release.yml
   # Drupal example: azure-pipelines-drupal-release.yml
   template: azure-pipelines-harrastukset-review.yml@harrastukset-pipelines
+  parameters:
+    # Application build arguments and config map values as key value pairs.
+    # Does not contain all buildArguments, the rest located in harrastukset-pipelines/harrastukset-variables-development.yml and harrastukset-variables-common.yml
+    # The values here will override the values defined in the harrastukset-pipelines repository
+    buildArgs:
+      NEXTJS_DISABLE_SENTRY: 1
+      NEXT_PUBLIC_ALLOW_UNAUTHORIZED_REQUESTS: 1
+      NEXT_PUBLIC_CMS_ORIGIN: https://harrastus.hkih.stage.geniem.io
+      NEXT_PUBLIC_LINKEDEVENTS_EVENT_ENDPOINT: https://api.hel.fi/linkedevents/v1/event
+      NEXT_PUBLIC_DEBUG: 1
+      NEXT_PUBLIC_MATOMO_ENABLED: 0
+    configMap: # pod environment variables
+      NEXT_PUBLIC_ALLOW_UNAUTHORIZED_REQUESTS: 1
+      NEXT_PUBLIC_CMS_ORIGIN: https://harrastus.hkih.stage.geniem.io
```

Comment:
the URL variables in both, the buildArgs and in the configMap, does not make sense imho.

Reply:
This is done as it is configured now. The ticket HH-299 is for dividing variables into build and runtime.